### DISCUSSION TITLE
Heatmap Uncolor/Hide cells with empty/null values

### DISCUSSION THREAD

**Discussion by iculda** [grafana/grafana#58517] [Dashboards, Panels & Visualizations]
Hello Is there any possibility to hide the cells with null values from a heatmap panel?
Thank you

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
